src/webxdc.rs

```diff
@@ -574,8 +576,15 @@ impl Context {
             bail!("receive_status_update: status message has no parent.")
         };
 
+        let chat_type: Chattype = self
+            .sql
+            .query_get_value("SELECT type FROM chats WHERE id=?", (chat_id,))
+            .await?
+            .with_context(|| format!("Chat type for chat {chat_id} not found"))?;
+
         if from_id != ContactId::SELF
             && !chat::is_contact_in_chat(self, instance.chat_id, from_id).await?
+            && chat_type != Chattype::Mailinglist
```

Comment:
`chat_type` is not needed for most updates so the sql-query is superfluous in most cases. we could maybe create a `get_chat_type()` function or so, so that the sql is executed only if the contact is not in chat.

or is that kind of optimisation already done automagically by rust? i doubt so, but not sure :)

no blocker, however